axlearn/experiments/text/gpt/fuji.py

```diff
@@ -188,6 +188,7 @@ def get_trainer_kwargs(
                 num_kv_heads=None if version == Version.V1 else 8,
                 rope_theta=rope_theta,
                 flash_attention=flash_attention,
+                remat_offload_dst="pinned_host",
```

Comment:
Is this option limited to 70B? Do we want to apply it to 7B and other models?

Reply:
Only use remat checkpoint offload into host when you could benefit from the extra TPU memory.

If there is plenty TPU memory then remat checkpoint into TPU memory.
If TPU memory is low or you want to squeeze as much per device batch as possible, then use offload_dst=pinned_Host.

So yes, it could make a lot of sense for 7B too on V5e and Trilium since then we can possibly increase per device batch size.

Reply:
I would prefer to have this PR focus on enabling remat offload and 70B. As a follow up I can do the following as part of V5E perf benchmarking:
* Enable remat offload for 7B and compare performance before and after
* See if I can increase per device batch size for 7B after enabling remat_offload

Would that work for you?

Reply:
Sounds good. Thanks.